src/vivarium_nih_us_cvd/components/observers.py

```diff
@@ -166,16 +166,13 @@ def setup(self, builder: Builder) -> None:
             builder.results.register_observation(
                 name=f"healthcare_visits_{visit_type}",
                 pop_filter=f'alive == "alive" and visit_type == "{visit_type}"',
-                aggregator=self.calculate_visit_counts,
+                aggregator=len,
```

Comment:
See https://github.com/ihmeuw/vivarium/blob/main/src/vivarium/framework/results/interface.py#L140. You don't need to define the aggregator at all.

Reply:
Oh yeah you even mentioned in your original comment. I'll remove this line and merge.